API/src/queries/orderQueries.js

```diff
@@ -0,0 +1,56 @@
+import orderData from '../db/dbData';
+
+class Data {
+  constructor(customer, recipientName, recipientAddr, recipientPhone, order) {
+    Data.count = Data.count ? Data.count + 1 : 1;
+
+    this.orderId = Data.count;
+    this.customer = customer;
+    this.recipientName = recipientName;
+    this.recipientAddress = recipientAddr;
+    this.recipientPhone = recipientPhone;
+    this.order = order;
+    this.orderTime = Date.now();
+    this.status = 'waiting';
+  }
+
+  static createNewOrder(recipientName, recipientAddr, recipientPhone, order) {
+    const customer = 'Adeolu James';
```

Comment:
Is this supposed to be hardcoded in this way?

Reply:
I plan to use authentication to get and validate the user from the database, but the routes created thus far does not contain creating a user (so I don't have users). Creating an order requires `customer`, I had to resort to this for the main time.

Reply:
Alright.